templates/modules/common/subscription-details.hypr.live

```diff
@@ -8,7 +8,15 @@
     <div>
         <div><strong> {{ labels.subscriptionNumber }}</strong> : {{ model.number }}</div><br/>
         <div><strong> {{ labels.subscriptionStatus }}</strong> : {{ model.status }} </div><br/>
-        <div><strong> {{ labels.frequency }}</strong>: {{model.frequency.value}} {{model.frequency.unit|lower}} </div><br/>
+        <div><strong> {{ labels.frequency }}</strong>: {{model.frequency.value}} 
+        {% if model.frequency.value > 1 %}
+            {% if model.frequency.unit|lower == "day" || model.frequency.unit|lower == "days" %} {{ labels.days }} {% endif %}
```

Comment:
get rid of these check's from all of the 3 if's `model.frequency.unit|lower == "days"` because units will be always singular like 'day', 'week'.